docs/current/requirements.md

```diff
@@ -55,15 +55,19 @@ The Vaquero project is designed to simplify the provisioning and ongoing operati
 20. Support incremental (multistep) cluster provisioning.
 21. Deploy a first build of all software clusters in lab.
 22. Separated Vaquero Server and Vaquero Agent to manage internal labs.
-
-
-### In Progress
-1. Manage power state of the hardware.
-2. Validate Vaquero with dual homed CleverSafe machines.
-3. Successfully guide multi-stage deployments managed by state machine on vaquero server(s).
-4. Provide state transition details and host status to provide operational insights.
-5. Upgrade workflow definitions to run pluggable pre-reboot and post-reboot jobs.
-
+23. Successfully guide multi-stage deployments managed by state machine on vaquero server(s).
+24. High availability Vaquero servers via Etcd
+25. Readiness probe for Kubernetes deployments and high availability
+26. Manage power state of the hardware
+27. Upgrade workflow definitions to run pluggable pre-reboot and post-reboot jobs.
+28. A policy engine that will ensure operational safety when updating a site, such as valid reboot times, minimum machines to be operational, dependency tracking and validation.
+29. A pluggable framework to enable pre-shutdown actions to be taken on a single host to flush its workload before cutting the power
+30. A pluggable framework to enable post-boot actions to validate a successful deployment of a single host.
+
+### On the Roadmap
+
+1. Provide state transition details and host status to provide operational insights (via userAPI)
+2. Liveness probe for Kubernetes
```

Comment:
we have this done, lets just remove it. Its not as large of a feature as the rest of the list, given its hardcoded 200 when alive